docker/scripts/dev_start.sh

```diff
@@ -424,9 +424,7 @@ function main(){
     fi
     set +x
 
-    # User with uid=1000 or username=apollo excluded
-    if [[ "${USER}" != "root" ]] && [[ "${USER}" != "apollo" ]] \
-        && [[ $USER_ID -ne 1000 ]]; then
+    if [[ "${USER}" != "root" ]]; then
```

Comment:
We may need to keep the `"${USER}" != "apollo"` part, as there is already a `apollo` user in the image, @storypku correct?

Reply:
here is a situation: if  default user_name of system is `apollo`, when he execute dev_start.sh, docker_adduser.sh will not be executed，then some devices such as ttyACM* will not get proper permissions

Reply:
sadly , now the default system user_name of D-kit is apollo

Reply:
> We may need to keep the `"${USER}" != "apollo"` part, as there is already a `apollo` user in the image, @storypku correct?

Yeap. So we should re-factore `docker_add_user.sh` asap to fix https://github.com/ApolloAuto/apollo/issues/11296